docsource/index.rst

```diff
@@ -0,0 +1,20 @@
+.. JGI/KBase ID Mapping Service documentation master file, created by
```

Comment:
Here is a sphinx docs theme that I recommend: https://github.com/rtfd/sphinx_rtd_theme

Reply:
Cool. If I ever put the docs up somewhere that looks like a good choice.